ethereum/messages.py

```diff
@@ -204,9 +212,8 @@ def _apply_casper_vote_transaction(state, tx):
     if tx.sender != null_address:
         state.increment_nonce(tx.sender)
 
-    # buy startgas
-    assert state.get_balance(tx.sender) >= tx.startgas * tx.gasprice
-    state.delta_balance(tx.sender, -tx.startgas * tx.gasprice)
+    # Let's give an upper bound for startgas
```

Comment:
Why this bound? Should this be part of the implementation guide?

Reply:
this is `STARTGAS` from `tester`. Since importing tester here causing an error, use a magic number instead. It's a bad practice, not sure if any way to make it better.

Reply:
Right, but do we need to run an assertion on start gas of these free-gas transactions at all? The only transactions that get passed into here are the initialize epoch tx and casper vote txs. They are all relatively bounded operations and because they are free, it doesn't really matter how much gas the tx sender specified

Reply:
That makes sense.